Runtime/LRS/Domain/Publisher.cs

```diff
@@ -252,13 +280,37 @@ public void SendCompletedStatement()
             public async void SendStatement(String verbId,
                                             String verbDisplay)
             {
+                Func <JsonObject, JsonObject> identity = (s) =>
+                {
+                    return s;
+                };
                 var statement = await FormBasicStatement(verbId,
                                                          verbDisplay,
                                                          user,
                                                          gameId,
                                                          gameDisplay,
-                                                         registrationIdentifier);
-                var statementStr = statement.Serialize();
+                                                         registrationIdentifier,
+                                                         identity);
+                InvokeStatementSent(statement);
+                var statementStr = statement.ToJsonString();
+                var response = await sender.SendStatement(statementStr);
```

Comment:
Is there a reason why you don't make repeated calls to `publisher.SendStatement` with an identity `StatementFn`, instead of writing out minor variations of the same function body?

Reply:
good point could dry it up here